zebra-consensus/src/parameters/subsidy.rs

```diff
@@ -33,11 +37,11 @@ pub const BLOSSOM_POW_TARGET_SPACING_RATIO: u32 = 2;
 /// Halving is at about every 4 years, before Blossom block time is 150 seconds.
 ///
 /// `(60 * 60 * 24 * 365 * 4) / 150 = 840960`
-pub const PRE_BLOSSOM_HALVING_INTERVAL: Height = Height(840_000);
+pub const PRE_BLOSSOM_HALVING_INTERVAL: HeightDiff = 840_000;
 
 /// After Blossom the block time is reduced to 75 seconds but halving period should remain around 4 years.
-pub const POST_BLOSSOM_HALVING_INTERVAL: Height =
-    Height(PRE_BLOSSOM_HALVING_INTERVAL.0 * BLOSSOM_POW_TARGET_SPACING_RATIO);
+pub const POST_BLOSSOM_HALVING_INTERVAL: HeightDiff =
+    PRE_BLOSSOM_HALVING_INTERVAL * (BLOSSOM_POW_TARGET_SPACING_RATIO as i32);
```

Comment:
Optional: this would be clearer.

```suggestion
    PRE_BLOSSOM_HALVING_INTERVAL * (BLOSSOM_POW_TARGET_SPACING_RATIO as HeightDiff);
```

Reply:
a765be43bf